src/keygen/keyshare.rs

```diff
@@ -42,6 +49,80 @@ impl KeySharePrivate {
     pub(crate) fn public_share(&self) -> Result<CurvePoint> {
         CurvePoint::GENERATOR.multiply_by_scalar(&self.x)
     }
+
+    /// Convert private material into bytes.
+    ///
+    /// 🔒 This is intended for use by the calling application for secure
+    /// storage. The output of this function should be handled with care.
+    pub fn into_bytes(self) -> Vec<u8> {
+        // Format:
+        // KEYSHARE_TAG | key_len in bytes | key (big endian bytes)
+        //              | 8 bytes          | key_len bytes
+
+        let share = self.x.to_bytes();
+        let share_len = share.len().to_le_bytes();
+
+        [KEYSHARE_TAG, &share_len, &share].concat()
```

Comment:
`share` seems to be private + sensitive data here. Should it be zeroized?

Reply:
Yes, good catch!